Backend/Schedules/Repository/IAppointmentRepository.cs

```diff
@@ -15,7 +15,9 @@ public interface IAppointmentRepository : ICreate<Appointment>, IGet<Appointment
    {
         Dictionary<int, Appointment> GetAppointmentsBy(DateTime date);
         Dictionary<int, Appointment> GetScheduledFromToday();
-
+        Appointment getAppointmentById(int id);
+        Appointment update(Appointment appointment);
+        IEnumerable<Appointment> GetAppointmentsByPatientId(string Id);
```

Comment:
On the other hand, every sql repository will inherit MySqlRepository class and in that class you can find methods like Update and GetObject, exact same methods you wanted to implement here.